.github/ISSUE_TEMPLATE/01_bug_report.yml

```diff
@@ -13,7 +13,7 @@ body:
           * Checked existing open *and* closed [issues](https://github.com/awesomeassertions/awesomeassertions/issues?utf8=%E2%9C%93&q=is%3Aissue), to see if the issue has already been reported?
           * Tried reproducing your problem in a new isolated project?
           * Read the [documentation](https://awesomeassertions.org/introduction)?
-          * Searched the two [test](https://github.com/awesomeassertions/awesomeassertions/tree/main/Tests/FluentAssertions.Specs) [suites](https://github.com/awesomeassertions/awesomeassertions/tree/main/Tests/FluentAssertions.Equivalency.Specs) if there is a test documenting the expected behavior?
+          * Searched the two [test](https://github.com/awesomeassertions/awesomeassertions/tree/main/Tests/AwesomeAssertions.Specs) [suites](https://github.com/awesomeassertions/awesomeassertions/tree/main/Tests/AwesomeAssertions.Equivalency.Specs) if there is a test documenting the expected behavior?
           * Considered if this is a general question and not a bug? For general questions please use [Stack Overflow](https://stackoverflow.com/questions/tagged/fluent-assertions?mixed=1).
```

Comment:
Should we also mention this here, that even if the search tag is "fluent-assertions" it can be used even for AA?